packages/client/src/views/UserSetup/CreatePassword.test.tsx

```diff
@@ -49,6 +78,32 @@ describe('CreatePassword page tests', () => {
     })
     component.find('button#Continue').simulate('click')
   })
+  it('it passes validations when Enter/Return key is pressed on ConfirmPassword field', () => {
+    component.find('input#NewPassword').simulate('change', {
+      target: { id: 'NewPassword', value: '0crvsPassword' }
+    })
+    component.find('input#ConfirmPassword').simulate('change', {
+      target: { id: 'ConfirmPassword', value: '0crvsPassword' }
+    })
+    component.find('input#ConfirmPassword').simulate('keyDown', {
+      key: 'Enter',
+      keyCode: 13,
+      which: 13
+    })
+  })
+  it('it passes validations when Enter/Return key is pressed on NewPassword field', () => {
+    component.find('input#NewPassword').simulate('change', {
+      target: { id: 'NewPassword', value: '0crvsPassword' }
+    })
+    component.find('input#ConfirmPassword').simulate('change', {
+      target: { id: 'ConfirmPassword', value: '0crvsPassword' }
+    })
+    component.find('input#NewPassword').simulate('keyDown', {
+      key: 'Enter',
+      keyCode: 13,
+      which: 13
+    })
+  })
```

Comment:
We should add in some assertions for these two as otherwise we are not really checking anything :sweat_smile: 
It could be something like "expect #GlobalError to not be present"

Reply:
I have added a check for "expect #GlobalError to not be present" as you recommended. Let me know if this suffices.

Reply:
I have done the same for the already existing test for passing validations when the Continue Button is clicked, starting on line 72, it too didn't have an assertion. I hope that is ok.

Reply:
That's fine. Thanks you!